lib/src/state/dialog.dart

```diff
@@ -25,6 +25,7 @@ abstract class Dialog with BuiltJsonSerializable implements Built<Dialog, Dialog
       {String title,
       Iterable<DialogItem> items,
       Iterable<Iterable<int>> mutually_exclusive_checkbox_groups = const [],
+      Iterable<int> force_disable = const {},
```

Comment:
I don't quite understand this `force_disable` option. Maybe it's needed, but if we can use the existing framework to enable/disable certain input fields, that's preferable. Maybe we can talk about on Wednesday why this is needed. (Or just explain in a reply here.)